lib/travis/build/script/shared/rvm.rb

```diff
@@ -15,30 +15,46 @@ module RVM
           rvm_remote_server_verify_downloads3=1
         )
 
+        DEFALUT_VERSION = 'default'
+
         def export
           super
-          sh.export 'TRAVIS_RUBY_VERSION', config[:rvm], echo: false if rvm?
+          # for now, we take the rvm key if both are defined
+          # see also:`#version` and `#version_manager`
+          if rvm?
+            sh.export 'TRAVIS_RUBY_VERSION', config[:rvm], echo: false
+          elsif config[:ruby]
+            sh.export 'TRAVIS_RUBY_VERSION', config[:ruby], echo: false
+          else
+            sh.export 'TRAVIS_RUBY_VERSION', DEFALUT_VERSION, echo: false
```

Comment:
Here, the Ruby version is picked up as `default` when neither `rvm` nor `ruby` key is present.